src/commands/Minion/minion.ts

```diff
@@ -442,7 +442,8 @@ Type \`confirm\` if you understand the above information, and want to become an
 
 			await msg.author.settings.update([
 				[UserSettings.Minion.Ironman, true],
-				[UserSettings.Minion.HasBought, true]
+				[UserSettings.Minion.HasBought, true],
+				[UserSettings.BitField, BitField.BypassAgeRestriction]
```

Comment:
This means ANYONE can bypass the age restriction by just becoming an iron then de-ironning.